src/main/java/mate.academy/service/impl/ProductParserImpl.java

```diff
@@ -1,14 +1,22 @@
 package mate.academy.service.impl;
 
 import java.math.BigDecimal;
+import mate.academy.lib.Component;
+import mate.academy.lib.Inject;
 import mate.academy.model.Product;
 import mate.academy.service.ProductParser;
 
+@Component
 public class ProductParserImpl implements ProductParser {
+    @Inject
     public static final int ID_POSITION = 0;
+    @Inject
     public static final int NAME_POSITION = 1;
+    @Inject
     public static final int CATEGORY_POSITION = 2;
+    @Inject
     public static final int DESCRIPTION_POSITION = 3;
+    @Inject
     public static final int PRICE_POSITION = 4;
```

Comment:
The `@Inject` annotation is incorrectly used on constants. The `@Inject` annotation is meant for fields that require dependency injection, not for constants. These constants should not be annotated with `@Inject`.